### PR TITLE
hyper_v: Change hv flags to 'all features' against 'all but one'

### PR DESCRIPTION
**Pull request by kimi1978wy**
change hv_time and hv_reset case to this format.

id:1840970

Signed-off-by: Yu Wang <wyu@redhat.com>

### CODE REVIEW

**Comment by kimi1978wy** [comment]
@YongxueHong Could you help to review it?

File: qemu/tests/hv_time.py [hunk 1]
```diff
@@ -117,8 +117,13 @@ def _check_result(cycles_without_flag, cycles_with_flag):
...
     timeout = params.get("timeout", 360)
-    flags_without_hv_time = params["flags_without_hv_time"]
-    flags_with_hv_time = params["flags_with_hv_time"]
+    hv_time_flags = params["hv_time_flags"].split()
+    flags_with_hv_time = params["cpu_model_flags"]
+    flags_without_hv_time = flags_with_hv_time.split(',')
+    for flag in hv_time_flags:
+        if flag in flags_without_hv_time:
+            flags_without_hv_time.remove(flag)
+    flags_without_hv_time = ','.join(flags_without_hv_time)
```
Comment (YongxueHong) [inline qemu/tests/hv_time.py:126]: For line120~126, how about?
```
    flags_without_hv_time = ','.join(
            [_ if _ not in hv_time_flags for _ in flags_with_hv_time.split(',')])
```

Comment (kimi1978wy) [reply qemu/tests/hv_time.py]: updated, and it seems that should be: 
```
    flags_without_hv_time = ','.join(
        [_ for _ in flags_with_hv_time.split(',') if _ not in hv_time_flags])
```

**Comment by YongxueHong** [comment]
Please upload the test results.
Thanks.

**Comment by kimi1978wy** [comment]
(01/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (780.16 s)
 (02/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (829.74 s)
 (03/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (976.78 s)
 (04/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hv_time.q35: PASS (369.29 s)
 (05/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hv_crash.q35: PASS (292.17 s)
 (06/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hv_reset.q35: PASS (148.11 s)
 (07/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.hv_time.q35: PASS (269.21 s)
 (08/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.hv_crash.q35: PASS (214.99 s)
 (09/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.hv_reset.q35: PASS (94.24 s)
 (10/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.hv_time.q35: PASS (264.10 s)
 (11/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.hv_crash.q35: PASS (211.76 s)
 (12/12) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.hv_reset.q35: PASS (110.73 s)
 (1/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.hv_time.q35: PASS (532.07 s)
 (2/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.hv_reset.q35: PASS (237.09 s)
 (3/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.hv_crash.q35: PASS (367.45 s)
 (4/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.hv_time.q35: PASS (496.78 s)
 (5/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.hv_reset.q35: PASS (212.95 s)
 (6/6) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.hv_crash.q35: PASS (333.36 s)

**Pull request merged**